src/main/java/org/apache/datasketches/quantiles/ItemsUnion.java

```diff
@@ -126,8 +137,8 @@ public void update(final ItemsSketch<T> sketchIn) {
    * @param srcMem Memory image of sketch to be merged
    * @param serDe an instance of ArrayOfItemsSerDe
    */
-  public void update(final Memory srcMem, final ArrayOfItemsSerDe<T> serDe) {
-    final ItemsSketch<T> that = ItemsSketch.getInstance(srcMem, comparator_, serDe);
+  public void union(final Memory srcMem, final ArrayOfItemsSerDe<T> serDe) { //TODO rename merge
```

Comment:
should we act on this now?

Reply:
It will make it consistent with our theta and tuple sketches.  Why not?

Reply:
no objections. I just saw this comment and thought that the time for breaking changes is now.